test/bbmustache_tests.erl

```diff
@@ -21,6 +21,17 @@ parse_file_test_() ->
      {"file_not_exist (with extension)",    ?_assertError(?FILE_ERROR, bbmustache:parse_file(<<"not_exist.mustache">>))}
     ].
 
+
+custom_serializer_test_() ->
+    [
+     {"simple function replacement",
+      fun() ->
+              ?assertEqual(<<"test, test">>,
```

Comment:
Please add or replace test cases.

- use two mustache (`{{ }}`) and rendered value escaped.
- use three mustache (`{{{ }}}`) and rendered value unescaped.

sample:
```erl
bbmustache:render(<<"{{s}} , {{{s}}}">>, [{"s", "A&B"}], [{value_serializer, fun(X) -> "<" ++ X ++ ">" end}]).
```